mavros/src/plugins/imu.cpp

```diff
@@ -145,85 +145,85 @@ class IMUPlugin : public plugin::PluginBase {
 	/**
 	 * @brief Fill and publish IMU data message.
 	 * @param time_boot_ms		Message timestamp (not syncronized)
-	 * @param orientation_enu	Orientation in the base_link ENU frame
+	 * @param orientation_nwu	Orientation in the base_link NWU frame
 	 * @param orientation_ned	Orientation in the aircraft NED frame
-	 * @param gyro_enu		Angular velocity/rate in the base_link ENU frame
+	 * @param gyro_nwu		Angular velocity/rate in the base_link NWU frame
 	 * @param gyro_ned		Angular velocity/rate in the aircraft NED frame
 	 */
-	void publish_imu_data(uint32_t time_boot_ms, Eigen::Quaterniond &orientation_enu,
-				Eigen::Quaterniond &orientation_ned, Eigen::Vector3d &gyro_enu, Eigen::Vector3d &gyro_ned)
+	void publish_imu_data(uint32_t time_boot_ms, Eigen::Quaterniond &orientation_nwu,
```

Comment:
everything in `publish_imu_data` is on ENU frame. Everything on `publish_imu_data_raw` is on NWU frame.